src/components/Books.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Book from './Book';
+import Inputs from './Inputs';
+
+const listBooks = [
+  { title: 'The Hunger Games', author: 'Suzanne Collins', category: 'Action' },
+  { title: 'Dune', author: 'Frank Herbert', category: 'Science Fiction' },
+  { title: 'Capital in the Twenty-First Century', author: 'Suzanne Collins', category: 'Economy' },
+];
+
+const Books = () => (
+  <div className="books-container">
+    <ul className="books">
+      <Book
+        title={listBooks[0].title}
+        author={listBooks[0].author}
+        category={listBooks[0].category}
+      />
+      <Book
+        title={listBooks[1].title}
+        author={listBooks[1].author}
+        category={listBooks[1].category}
+      />
+      <Book
+        title={listBooks[2].title}
+        author={listBooks[2].author}
+        category={listBooks[2].category}
+      />
```

Comment:
Considering that you might not always know the number of elements in the array and also to make the code DRYer, I suggest that you use a map statement here.

e.g:

```
     {listBooksooks.map((book) => (
            <Book
              title={book.title}
              author={books.author}
              category={book.category}
            />
      ))}
```

Reply:
Hi @emmiiorji , thank you for the tip! I'll go ahead and make the change 😄